sequencer/src/context.rs

```diff
@@ -126,8 +126,8 @@ impl<N: ConnectedNetwork<PubKey>, P: SequencerPersistence, V: Versions> Sequence
             .load_consensus_state::<V>(instance_state.clone())
             .await?;
 
-        let stake_table_commit =
-            compute_stake_table_commitment(&config.known_nodes_with_stake, stake_table_capacity)?;
+        let stake_table: HSStakeTable<SeqTypes> = config.known_nodes_with_stake.clone().into();
```

Comment:
can we instead have some method like: 
`config.hotshot_stake_table() -> HSStakeTable` instead? because most people probably don't know there's a conversion from `Vec<PeerConfig>` to that.